src/Communication/Message.php

```diff
@@ -82,7 +82,7 @@ public function __toString(): string
         return json_encode([
             'id'        => $this->getId(),
             'method'    => $this->getMethod(),
-            'params'    => $this->getParams()
+            'params'    => empty($this->getParams())?new \stdClass():$this->getParams()
```

Comment:
Surely the proper fix here is to cast `$this->getParams()` to an `object`?

Reply:
now (2 months later) I don't remember exactly, but there were occurences where an array was needed, I'm not sure where exactly